server/src/main/java/com/epam/aidial/core/server/config/FileConfigStore.java

```diff
@@ -195,4 +198,17 @@ private static void setMissingFeatures(Deployment model, Features features) {
             modelFeatures.setContentPartsSupported(features.getContentPartsSupported());
         }
     }
+
+    private JsonMapper buildJsonMapper(JsonObject settings) {
+        JsonMapper mapper = ProxyUtil.MAPPER.copy();
+
+        boolean overwriteArrays = settings
+                .getJsonObject("jsonMergeStrategy", new JsonObject())
+                .getBoolean("overwriteArrays", false);
+
+        mapper.configOverride(ArrayNode.class)
+                .setMergeable(!overwriteArrays);
```

Comment:
It is a matter of preference. But I would rename the variable to `mergeArrays`, invert the logic and get rid of that negation. For me it just sounds more naturally. 
I don't insist.

Reply:
While I do agree with you as a developer, from user perspective, imho, `overwriteArrays` would be better as it gives more precise information about what will happen (merge is too abstract in my opinion, I didn't even know prior to this case that "merging" in terms of arrays is "concat").

Reply:
Let's wait for @astsiapanay and his opinion, and then decide. I don't have any strong objections about renaming just in case.

Reply:
I agree with @PMitrafanau let's make a favor of the user perspective. That makes the property's meaning more simple.
`mergeArrays` may confuse an user: at least what strategy the Core will apply in two cases keep unanswered.